README.md

```diff
@@ -20,7 +20,7 @@
 
 ## What is it?
 
-**pandas** is a Python package providing fast, flexible, and expressive data
+**pandas** is a Python package that provides fast, flexible, and expressive data
```

Comment:
This sentence is also in LONG_DESCRIPTION of setup.py. should probably keep in sync so can you change there as well?

Reply:
Got it. I made the changes. It's being checked right now.

Reply:
Hey! I've made the necessary changes.